src/pages/library/Datasets.js

```diff
@@ -294,9 +294,9 @@ const hca = () => h(Participant, {
   monitoring, and treating disease.`
 }, [
   h(ButtonPrimary, {
-    disabled: true,
-    tooltip: 'HCA not yet in production'
-    //when in production, add this tooltip: browseTooltip
+    href: 'https://data.humancellatlas.org/explore/projects',
+    target: '_blank',
+    tooltip: 'Look for the export selected data icon to export data from this provider.'
```

Comment:
The text says icon, but it is in fact a button, so we should say so:
```suggestion
    tooltip: 'Look for the Export Selected Data button to export data from this provider.'
```
(capitalization is theirs)